pkgs/development/python-modules/decora_wifi/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, python313Packages }:
```

Comment:
You can take `requests` here directly, and it'll automatically come from the right Python version in the set. Also, extremely nitpicky style thing, but you might want to put every argument on a separate line, just to reduce diff sizes in the future.

Reply:
Actually, nixfmt should just do the right thing here.